common/djangoapps/student/forms.py

```diff
@@ -234,3 +235,28 @@ def cleaned_extended_profile(self):
             for key, value in self.cleaned_data.items()
             if key in self.extended_profile_fields and value is not None
         }
+
+
+def dropdown_context():
```

Comment:
It's not clear from the method name or the docstring that this method is for the registration form specifically.

Also per [edX Python guidelines](https://github.com/edx/edx-platform/wiki/Python-Guidelines) the return value would be better as a `namedtuple`.